docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -40,8 +40,6 @@
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
 <h2>Release notes: Open GEE 5.3.1</h2>
 
-<p>Open GEE 5.3.1 is currently in development</p>
-
   <h5>
     <p id="overview_5.3.1">New Features</p>
```

Comment:
Can we make these section headers larger? Maybe h3 or h4.

Reply:
ok